### ISSUE TITLE
Click makes options wrap by single characters; options are difficult to read

### ISSUE THREAD

**Issue by JTFouquier** [JTFouquier/ghost-tree#13]
Click works great, but it still wraps the command options by single characters.  Sometimes there are 4 to 5 options in ghost-tree, making reading options very difficult.  I’ve asked Click how the bug fix is going, and they have a workaround but haven't accepted it as a PR. Can ghost-tree be packaged with a modified version of Click (not released by Click)?

Click Issue:
https://github.com/mitsuhiko/click/issues/231  (they gave me directions on how to implement the PR, but I'm not sure it's worth trying due to future packaging of ghost-tree)
Click PR:
https://github.com/mitsuhiko/click/pull/240

According to @gregcaporaso via previous email (3/26/15):
"it's better to just leave as-is then try to build a modified version of click"

Example:

![screen shot 2015-03-29 at 9 00 45 pm](https://cloud.githubusercontent.com/assets/7920357/6890006/d9d52478-d656-11e4-87c0-7b21204fee01.png)

**Comment by jairideout** [comment]
Since someone is working on a fix in click, it might be worth waiting to see if this gets resolved before an initial ghost-tree release. If it isn't fixed by then, @ebolyen or I can help with packaging a modified version of click with ghost-tree (should be possible because click is BSD-licensed).

**Comment by JTFouquier** [comment]
I think they were considering the PR for a while. Looks like they merged it last night.  So does that mean that the next Click release will contain the fix? I suppose it should if they pulled it.  Thanks!

**Comment by jairideout** [comment]
Yep, the next click release should contain the fix. When ghost-tree is ready for an initial release, let's check to see if there's been a click release. If not, we can package the latest dev version of click with ghost-tree.

**Comment by JTFouquier** [comment]
@ebolyen the Click fixes are referenced above.  I don't think they made it into the new release of Click 4.0 (http://click.pocoo.org/4/changelog/#version-4-0).

**Comment by jairideout** [comment]
Looks like they fixed it in click 4.0.

With click 3.3:

```
$ ghost-tree scaffold hybrid-tree --help
Usage: ghost-tree scaffold hybrid-tree [OPTIONS] TIPS_OTU_TABLE
                                       T
                                       I
                                       P
                                       S
                                       _
                                       T
                                       A
                                       X
                                       O
                                       N
                                       O
                                       M
                                       Y
                                       _
                                       F
                                       I
                                       L
                                       E
                                       T
                                       I
                                       P
                                       S
                                       _
                                       S
                                       E
                                       Q
                                       U
                                       E
                                       N
                                       C
                                       E
                                       _
                                       F
                                       I
                                       L
                                       E
                                       B
                                       A
                                       C
                                       K
                                       B
                                       O
                                       N
                                       E
                                       _
                                       A
                                       L
                                       I
                                       G
                                       N
                                       M
                                       E
                                       N
                                       T
                                       _
                                       F
                                       I
                                       L
                                       E
                                       G
                                       H
                                       O
                                       S
                                       T
                                       _
                                       T
                                       R
                                       E
                                       E
                                       _
                                       O
                                       U
                                       T
                                       P
                                       U
                                       T
                                       _
                                       F
                                       I
                                       L
                                       E

Options:
  --help  Show this message and exit.
```

With click 4.0:

```
$ ghost-tree scaffold hybrid-tree --help
Usage: ghost-tree scaffold hybrid-tree [OPTIONS] TIPS_OTU_TABLE
                                       TIPS_TAXONOMY_FILE TIPS_SEQUENCE_FILE
                                       BACKBONE_ALIGNMENT_FILE
                                       GHOST_TREE_OUTPUT_FILE

Options:
  --help  Show this message and exit.
```

I'll update setup.py to require click >= 4.0 in my open PR (#18) since I already have other fixes to ghost-tree's dependency versions.
